packages/flutter/test/widgets/binding_test.dart

```diff
@@ -131,9 +131,5 @@ void main() {
     await tester.pump(const Duration(milliseconds: 1));
     expect(tester.binding.hasScheduledFrame, isFalse);
     expect(frameCount, 0);
-
-    tester.binding.scheduleWarmUpFrame(); // this actually tests flutter_test's implementation
-    expect(tester.binding.hasScheduledFrame, isFalse);
-    expect(frameCount, 1);
```

Comment:
?

Reply:
testerbinding is no longer overriding scheduleWarmUpFrame, there is no point to test it?